### DISCUSSION TITLE
Issue when using different directory such as src

### DISCUSSION THREAD

**Discussion by Emt-tz** [jellydn/next-swagger-doc#827] [Q&A]
### Bug descrition

#When default root path is set to src 

```
"paths": {
      "@/*": [
        "./src/*"
      ]
    },
```

Swagger does not load the api routes even when annotated but when i move my code out of src directory then swagger does display

### Reproduction

use src directory as main file directory for the apis

### System Info

```Shell
macos ventura 13.2
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by jellydn** [comment]
Could you provide the reproduce repository? You would see that we could define the path for api routes.

```ts
const spec: Record<string, any> = createSwaggerSpec({
    apiFolder: 'pages/api' // or 'src/pages/api', ==> You could change the folder here
    definition: {
      openapi: '3.0.0',
      info: {
        title: 'Next Swagger API Example',
        version: '1.0',
      },
    },
  });
```
